set.mm

```diff
@@ -496507,6 +496553,132 @@ The sumset operation can be used for both group (additive) operations and
+        eqtrd 3eqtr3d ecref fveqeq2 adantl rspcedvd impbida velsn bitr4di bitrd
+        w3a eqcomd eqrdv fvex unisn eqtrdi ) AHDGUCQZUDZFRCYQUEZUFZHCRZAJYQCJUG
+        ZUEZUFYSBUHRZFUIOUUAYQSUUBYRUUAYQCUJUKAYQDUHRZYPULZUUCAHUUDTZYQUUETPUUD
+        HYPDGUCUMUNUOAYPDBUUDUIUPBDYPUQQSANURAUUDUSADGUCVEACDEUTQZTZDUPTZLDECVA
+        ZUOVBVCAYRUIACYQUUGLVDVFVGAYSYTVHZUFYTAYRUUKAUAYRUUKAUAUGZYRTUBUGZCRZUU
+        LSZUBYQVIZUULUUKTZAUBUUDYQUULCAUUDEUHRZCAUUHUUDUURCVRZLDECUUDUURUUDVJZU
+        URVJZVKZUOVLZAHYPUUDAGDVMRZTZGDVNRTZUUDYPVOZAGCVPIVHUEZUVDMAUUHUVHUVDTL
+        DECIKVQUOVSZGDVTZYPDUUDGUUTYPVJZWAWBZWIZWCAUUPUULYTSZUUQAUUPUVNAUUOUVNU
+        BYQAUUMYQTZWDZUUOWDUUNUULYTUVPUUOWEUVPUUNYTSUUOUVPYTHDWFRZRZCRZUUNEWGRZ
+        QZUVTQZYTIUVTQZUUNYTUVPUWAIYTUVTUVPUVRUUMDWGRZQZCRZUWAIUVPUUHUVRUUDTUUM
+        UUDTZUWFUWASAUUHUVOLWHZUVPUUDDUVQHUUTUVQVJZUVPUUHUUIUWHUUJUOZAUUFUVOPWH
+        ZWJAYQUUDUUMUVMWKZUWDUVTDEUVRCUUMUUDUUTUWDVJZUVTVJZWLWMUVPUWEUUDTZUWFIS
+        ZUVPCUUDWNZUWEUVHTZUWOUWPWDZAUWQUVOUVCWHUVPUWEGUVHUVPUUIGUUDWOZHUUMYPWP
+        ZUWEGTZUWJAUWTUVOAUVEUVFUWTUVIUVJUUDGDUUTWQWBWHAUUFUVOUXAPUUFUVOUXAUUMU
+        ITUUFUVOUXAWRUBXAUUMHYPUIUUDWSXBWTXKUUIUWTWDZUXAWDUUFUWGUXBUXCUXAUUFUWG
+        UXBYJHUUMUWDYPGDUVQUPUUDUUTUWIUWMUVKXCWTXDXEMXFUWQUWRUWSUUDIUWECXGWTXHX
+        IXJXLUVPUWBYTYTEWFRZRZUUNUVTQZUVTQZUUNUVPUWAUXFYTUVTUVPUVSUXEUUNUVTUVPU
+        UHUUFUVSUXESUWHUWKUUDDECUVQUXDHUUTUWIUXDVJZXMXHXNXLUVPEUPTZYTUURTZUUNUU
+        RTUXGUUNSUVPUUHUXIUWHDECXOUOZUVPUUDUURHCUVPUUHUUSUWHUVBUOZUWKXPZUVPUUDU
+        URUUMCUXLUWLXPUURUVTEUXDYTUUNUVAUWNUXHXQWMXTUVPUXIUXJUWCYTSUXKUXMUURUVT
+        EYTIUVAUWNKXRXHYAWHXJXSAUVNWDZUUOYTUULSZUBHYQAHYQTZUVNAUVGUUFUXPUVLPHYP
+        UUDYBXHWHUUMHSUUOUXOWRUXNUUMHUULCYCYDUXNUULYTAUVNWEYKYEYFUAYTYGYHYIYLUK
+        YTHCYMYNYOXT $.
+    $}
+
+    ${
+      ghmquskerlem2.y $e |- ( ph -> Y e. ( Base ` Q ) ) $.
+      $( Lemma for ~ ghmqusker (Contributed by Thierry Arnoux, 14-Feb-2025.) $)
+      ghmquskerlem2 $p |- ( ph -> E. x e. Y ( J ` Y ) = ( F ` x ) ) $=
+        ( co cfv wcel cv cqg cec wceq cbs wrex cqs cvv cgrp cqus a1i eqidd cghm
+        ovexd ghmgrp1 syl qusbas eleqtrrd elqsg biimpa syl2anc wa wer csubg csn
+        ccnv cima cnsg ghmker nsgsubg 3syl eqeltrid eqger ad2antrr simplr ecref
+        eqid simpr fveq2d ghmquskerlem1 eqtrd jca expl reximdv2 mpd ) AIBUAZEHU
+        BRZUCZUDZBEUESZUFZIGSZWFDSZUDZBIUFAICUESZTZIWJWGUGZTZWKQAIWOWQQAWGECWJU
+        HUICEWGUJRUDAOUKAWJULAEHUBUNADEFUMRTZEUITMEFDUOUPUQURWPWRWKBWJIWGWOUSUT
+        VAAWIWNBWJIAWFWJTZWIWFITZWNVBAWTVBZWIVBZXAWNXCWFWHIXCWJWGVCZWTWFWHTAXDW
+        TWIAHEVDSZTXDAHDVFJVEVGZXENAWSXFEVHSTXFXETMEFDJLVIXFEVJVKVLWGEWJHWJVQWG
+        VQVMUPVNAWTWIVOZWFWGWJVPVAXBWIVRZURXCWLWHGSWMXCIWHGXHVSXCCDEFGHWFJKLAWS
+        WTWIMVNNOPXGVTWAWBWCWDWE $.
+    $}
+
+    ghmqusker.s $e |- ( ph -> ran F = ( Base ` H ) ) $.
+    $( The mapping ` J ` induced by a surjective group homomorphism ` F ` from
```

Comment:
I find this a bit hard too read, here's my suggestion on how to reword it (feel free to ignore it or use parts for inspiration):
```
A surjective group homomorphism ` F ` from ` G ` to ` H ` induces an isomorphism ` J ` from ` Q ` to ` H ` ,
where ` Q ` is the factor group of ` G ` by ` F ` 's kernel ` K ` .
```

Reply:
Your description is much clearer IMO, I'll use it.